src/backend/db/categories.js

```diff
@@ -4,33 +4,44 @@ import { v4 as uuid } from "uuid";
  * Category Database can be added here.
  * You can add category of your wish with different attributes
  * */
+/*
+   <iframe width="697" height="392" src="https://www.youtube.com/embed/PKZABq7Llzs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+*/
 
 export const categories = [
   {
     _id: uuid(),
     categoryName: "flower",
-    image: "",
+    url:"https://youtu.be/PKZABq7Llzs",
+    iframe: "" ,
+    image: "https://i.ytimg.com/an_webp/PKZABq7Llzs/mqdefault_6s.webp?du=3000&sqp=CMukipMG&rs=AOn4CLDU_4QTTt2cDXvPY1uuqP6sy3-H-g",
     description:
       "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
   },
   {
     _id: uuid(),
     categoryName: "indoor",
-    image: "",
+    url:"https://youtu.be/kWr8SvbUGMo",
```

Comment:
Suggestion: You don't need to write the whole URL just id of that video would also be fine, and store youtube in some variable.